src/main.rs

```diff
@@ -95,6 +96,175 @@ fn get_issue_tree_init_hashes(repo: &Repository, _: &clap::ArgMatches) -> i32 {
     0
 }
 
+/// show subcommand implementation
+fn show(repo: &Repository, sub: &clap::ArgMatches) -> i32 {
+    let issue = match sub.value_of("parent") {
+        Some(s) => s,
+        None => {
+            error!("No parent hash");
+            return 1;
+        },
+    };
```

Comment:
If configured correctly, clap should generate an error. An `unwrap` should be sufficient.